tutorials/super_blink/blink_config.h

```diff
@@ -0,0 +1,31 @@
+#include "libs/can/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+#include "libs/timer/api.h"
+
+void timer0_callback(void);
+
+gpio_t LED1 = PB0;
+gpio_t LED2 = PB1;
+
+timer_cfg_s timer0_cfg = {
+    .timer = TIMER0,
+    .timer0_mode = TIMER0_MODE_CTC,
+    .prescalar = CLKIO_DIV_64,
+    .channel_a = {
+        .output_compare_match = 0x822,
```

Comment:
Timer 0 is an 8-bit timer, so it will count up to a maximum of `255`. That means when you use `0x822` as the compare match, it will actually be `0x822 & 0xFF` or `0x22`. So your frequency will be ~1838 Hz. You can either switch to timer 1 or you can use a different prescalar/output_compare_match pair.